pkg/server/cstorvolumeconfig/backup_endpoint.go

```diff
@@ -0,0 +1,544 @@
+/*
+Copyright 2020 The OpenEBS Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cstorvolumeconfig
+
+import (
+	"net/http"
+
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	cstorapis "github.com/openebs/api/pkg/apis/cstor/v1"
+	openebsapis "github.com/openebs/api/pkg/apis/openebs.io/v1alpha1"
+	cstortypes "github.com/openebs/api/pkg/apis/types"
+	clientset "github.com/openebs/api/pkg/client/clientset/versioned"
+	"github.com/openebs/api/pkg/util"
+	snapshot "github.com/openebs/cstor-operators/pkg/snapshot"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	k8serror "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/klog"
+)
+
+// backupAPIOps holds the clients, http request and
+// response
+type backupAPIOps struct {
+	req          *http.Request
+	resp         http.ResponseWriter
+	k8sclientset kubernetes.Interface
+	clientset    clientset.Interface
+	snapshoter   snapshot.Snapshoter
+}
+
+var (
+	// openebsNamespace is the namespace where openebs is deployed
+	openebsNamespace string
+)
+
+// backupV1alpha1SpecificRequest deals with backup API requests
+func (s *HTTPServer) backupV1alpha1SpecificRequest(resp http.ResponseWriter, req *http.Request) (interface{}, error) {
+	backupOp := &backupAPIOps{
+		req:          req,
+		resp:         resp,
+		k8sclientset: s.cvcServer.kubeclientset,
+		clientset:    s.cvcServer.clientset,
+		snapshoter:   s.cvcServer.snapshoter,
+	}
+
+	switch req.Method {
+	case "POST":
+		return backupOp.create()
```

Comment:
Let us log for create request also.

Reply:
Instead of logging here good to log in `create` method.

Reply:
Then why it is logged for `Get` and `Delete`

Reply:
I will log it.